spec/models/business_day_spec.rb

```diff
@@ -22,6 +22,16 @@
       expect(business_day.date).to eq(Date.parse('2022-12-30'))
     end
 
+    context 'when date_zero is given as a time' do
+      let(:day_zero) { '2023-07-07T23:04:35.351Z' }
+      let(:age_in_business_days) { 0 }
+
+      it 'converts the time to the corresponding date in TZ London' do
```

Comment:
Could this it block be more specific and say something like - "conversts the time to the correct next working day in TZ London"? Or am I misunderstanding this?